### ISSUE TITLE
CI: read_clipboard / to_clipboard not working in Azure

### ISSUE THREAD

**Issue by datapythonista** [pandas-dev/pandas#26887]
The clipboard system does not work in Azure Pipelines. I tried the next things:

Installing `xsel` and `xclip` (separately) as suggested in the documentation

Setting up a step in the azure settings based on the travis docs:
```
  - script: |
      set -x
      sudo apt-get install -y xsel xvfb
      export DISPLAY=":99.0"
      sh -e /etc/init.d/xvfb start
      sleep 3
    displayName: 'Setup clipboard'
```

But we still get the same paperclyp error. This is affecting the docs (see #26852) and also the tests (see #26428).

**Comment by datapythonista** [comment]
No idea what's wrong with the clipboard with azure, but managed to get this error:
```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-vsts'
qt.qpa.screen: QXcbConnection: Could not connect to display :99.0
Could not connect to any X display.
```

**Comment by jreback** [comment]
@datapythonista IIRC you have this working now?

**Comment by datapythonista** [comment]
There are problems with the second pytest call slow / not-slow. Shouldn't be a problem after the refactoring of the parallelization of the tests, but there are problems there with the timings I still couldn't figure out why are happening.

**Comment by jorisvandenbossche** [comment]
Removing the 1.0 milestone here, since we are still testing it on travis, I don't think this is a blocker for 1.0

**Comment by TomAugspurger** [comment]
@datapythonista is the original description still accurate? Anything we should be doing here for 1.1?

**Comment by datapythonista** [comment]
This was fixed in #29712, we are running the clipboard tests in some Azure builds now. Closing.
